twitchio/ext/sounds/__init__.py

```diff
@@ -43,13 +43,20 @@
 
 AP = TypeVar("AP", bound="AudioPlayer")
 
-has_ffmpeg: bool
+ffmpeg_bin: str = None
```

Comment:
A string can't be None, this should be Optional[str]

Reply:
It's done.